dev/io.openliberty.org.apache.myfaces.4.0_fat/test-applications/MYFACES-4658.war/src/io/openliberty/org/apache/faces40/fat/myfaces4658/ConfirmBehavior.java

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
```

Comment:
Should this be 2024 or was it copied from another test?

Reply:
Oops. should be 2024